index.html

```diff
@@ -141,12 +179,14 @@ <h2 class="myself">Hello I’m a software developer!
                 </ul>
             </div>
                     <button class="resume_button">Get my resume</button>
-                    <ul class="skills">
+            </section>
+                <ul class="skills">
                     <li class="knowledge first_knowledge">Languages<img class="arrow_down" src="images/rightarrow.png" alt="down arrow"></li>
                     </hr>
                         <li><img class="lang_img" src="images/js.png" alt="javaScript"></li>
                         <li><img class="lang_img" src="images/html.png" alt="html"></li>
                         <li><img class="lang_img" src="images/css.png" alt="css"></li>
+                        <li><img class="desktop_list" src="images/languagelist.png" alt="language list"></li>
```

Comment:
- [optional] Great job. Your contact section has a space/margin on the right. Please fix that so it follows the Figma design. Remember, this is only optional because it was not mentioned by the previous reviewer. Endeavor to apply it as it is important. 
yours
![Screenshot (771)_LI](https://user-images.githubusercontent.com/65298048/213659800-243e99bb-aba8-4536-bd4b-c362e4151d6f.jpg)
figma
![Screenshot (773)](https://user-images.githubusercontent.com/65298048/213659819-5942876c-64ca-4f3a-bed0-319f68ed6d29.png)

Reply:
@tufoinnkuo10 fixed! Thanks for the observation 👀 

![image](https://user-images.githubusercontent.com/108192399/213700210-4b26ba62-d028-4c36-8b1b-90562b919386.png)